lib/index.js

```diff
@@ -126,11 +126,10 @@ function* createEverything() {
 
   var adminTestDatabaseClient = pgp(adminTestDatabase);
   yield adminTestDatabaseClient.task(alterPrivileges);
-
-  pgp.end();
```

Comment:
P.S. Since it is a run-through process, you could, of course, run it with just a single exclusive connection, and thus not use the connection pool at all, but that's not necessarily better.

For that you would do `db.connect({direct: true})` in the beginning, and then do `connection.done()` to close it, and then do `db.connect({direct: true})` again. This would fully control your single connection. See method [Database.connect](http://vitaly-t.github.io/pg-promise/Database.html#.connect).